src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -823,47 +810,63 @@ private ImmutableArray<Symbol> CreateMembers()
                             if (tupleFieldIndex >= 0)
                             {
                                 // This is a tuple backing field
+
+                                // adjust tuple index for nesting
+                                if (currentNestingLevel != 0)
+                                {
+                                    tupleFieldIndex += (RestPosition - 1) * currentNestingLevel;
+                                }
+
+                                var providedName = _elementNames.IsDefault ? null : _elementNames[tupleFieldIndex];
+                                var location = _elementLocations.IsDefault ? null : _elementLocations[tupleFieldIndex];
+                                var defaultName = TupleMemberName(tupleFieldIndex + 1);
+                                // if provided name does not match the default one, 
+                                // then default element as declared implicitly
```

Comment:
Should this be `default element is ...`? Same question for other copies of this comment.